src/client/scripts/loginModal.js

```diff
@@ -12,6 +12,20 @@ Template.loginModal.events({
         eve.preventDefault();
         Session.set("author", eve.target.alias.value);
         $("#LogInModal").modal("hide");
-        SnackbarMethods.DisplayMessage("Alias set" , DEFAULT_SNACKBAR_TIMEOUT);
+        SnackbarMethods.DisplayMessage("Alias set", DEFAULT_SNACKBAR_TIMEOUT);
+        if (Session.get("roomCode")) {
```

Comment:
Personal preference, but this section just looks... nasty.... I would recommend just going through and cleaning up the logic a little bit, it's a bit hard to follow for something so simple.
Couple of clean code suggestions
1. Positive Case first (i.e. Session.get("aliasID") { Meteor.call(.......); } else {
2. Try to avoid nesting, you can rethink the logic, or extract out the functionality to a nicely named function, up to you
3. Avoid unnecessary braces (i.e. neither of those if statements inside the callback need braces)